daemon/src/common/Defs.h

```diff
@@ -42,6 +42,8 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 /** Default base package */
 #define DEFAULT_BASE_PAK    PRODUCT_NAME_LOWER
 
+#define DAEMON_FS_LEGACY_PAKS false
```

Comment:
Eventually all of these flags will be overridable in cmake, making this easier to specify